cmd/snap/cmd_export_key.go

```diff
@@ -0,0 +1,59 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/snapcore/snapd/asserts"
+	"github.com/snapcore/snapd/i18n"
+)
+
+type cmdExportKey struct {
+	Positional struct {
+		KeyName string `positional-arg-name:"<key-name>" description:"name of key to export"`
+	} `positional-args:"true" required:"true"`
+}
+
+func init() {
+	cmd := addCommand("export-key",
+		i18n.G("Export key"),
+		i18n.G("Export a public key in the format required by assertions."),
```

Comment:
It's the body of an assertion, not the whole thing (which would require further information such as the account-id).  But I'll rephrase it to "Export a public key assertion body ..." etc.